src/Entity/Transformer.php

```diff
@@ -181,8 +181,8 @@ public function loadBlock(array $content, PageInterface $page): PageBlockInterfa
 
         $block->setEnabled($content['enabled']);
 
-        if (isset($content['position'])) {
-            $block->setPosition($content['position']);
+        if (isset($content['position']) && is_numeric($content['position'])) {
+            $block->setPosition(\is_string($content['position']) ? ((int) $content['position']) : $content['position']);
```

Comment:
What about always casting to `int` ?

Reply:
I will try, I tried to be clever so I don't get complains for phpstan/psalm , for already int casting.